appinterface/projection/crossfire/crossfire.go

```diff
@@ -64,6 +66,7 @@ func NewCrossfire(
 		participantsListUrl:      participantsListURL,
 		rdbConn:                  rdbConn,
 		logger:                   logger,
+		primaryAddressPrefix:     primaryAddressPrefix,
```

Comment:
This line should better move to the line under `validatorAddressPrefix` to keep consistent with struct